azuredevops/internal/service/policy/branch/resource_branchpolicy_min_reviewers.go

```diff
@@ -44,6 +44,12 @@ func ResourceBranchPolicyMinReviewers() *schema.Resource {
 			ConflictsWith: []string{"settings.0.on_push_reset_approved_votes", "settings.0.on_push_reset_all_votes"},
 		},
 
+		"on_each_iteration_require_vote": {
```

Comment:
Does this conflict with other attributes, e.g. `on_last_iteration_require_vote`?